src/api/api.types.tsx

```diff
@@ -217,3 +217,23 @@ export interface Item
   usage_status: string;
   properties: Property[];
 }
+
+// ------------------------------------ ATTACHMENTS ------------------------------------------------
+
+export interface AttachmentsPostMetadata {
```

Comment:
```suggestion
export interface AttachmentPost {
```
Could be slightly confusing? But would be inline with the backend like the rest. Its a post to the `/attachments` endpoint not a post of a file which is what the object store one itself would be.